web/libs/editor/src/components/ImageView/SuggestionControls.js

```diff
@@ -84,14 +84,22 @@ export const SuggestionControls = observer(({ item, useLayer }) => {
           cornerRadius={16}
         />
         <ControlButton
-          onClick={() => item.annotation.rejectSuggestion(item.id)}
+          onClick={() => {
+            const ann = item.annotation;
+            ann.rejectSuggestion(item.id);
+            ann.deleteAllDynamicregions(true);
+          }}
```

Comment:
Most probably this is an unintentional change. It changes the current logic instead of fixing LEAP-942. 

When we work with a single suggestion we do not want to affect all other as we may not have a resolution of what we want to do with them on this action. Right now it became duplicate of rejectAllSuggestions which shouldn't be the case.

Reply:
It's intentional and follows the following logic:
- dynamic region is always removed after it was sent to the ML Backend
- therefore there could not be the case where you have more than a single dynamic region on the screen unless you use a special tool

Does it match your expectation?

Reply:
After conversation with Andrew internally it was indeed identified as not expected and we plan to revert it back.